app/src/main/java/edu/grinnell/grinnell_publications_android/Fragments/ArticleFragment.java

```diff
@@ -0,0 +1,162 @@
+package edu.grinnell.grinnell_publications_android.Fragments;
+
+
+import android.support.design.widget.CollapsingToolbarLayout;
+import android.os.Bundle;
+import android.support.design.widget.FloatingActionButton;
+import android.support.v4.app.Fragment;
+import android.util.DisplayMetrics;
+
+import android.support.v7.widget.Toolbar;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+import com.squareup.picasso.Picasso;
+import edu.grinnell.grinnell_publications_android.R;
+
+
+
+import java.util.Date;
+
+/**
+ * TODO: Fix getDrawable deprication
+ *
+ */
+
+public class ArticleFragment extends Fragment {
+
+    CollapsingToolbarLayout collapsingToolbar;
+    Toolbar toolbar;
+
+    private ImageView imageView;
+    private FloatingActionButton inviteButton;
+
+
+
+    ArticleFragment context = this;
+
+
+    /**
+     * Mandatory empty constructor for the fragment manager to instantiate the
+     * fragment (e.g. upon screen orientation changes).
+     */
+    public ArticleFragment() {
+    }
+
+    /**
+     * Constructor created by the newInstance and takes in the Item ID
+     */
+    public static ArticleFragment newInstance(){
+        ArticleFragment fragment = new ArticleFragment();
+        return fragment;
+    }
+    /*
+        public static DisplayFragment newInstance(Article article){
+        DisplayFragment fragment = new DisplayFragment();
+        return fragment;
+    }
+     */
+
+    /**
+     * OnCreate for the fragment
+     */
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+    }
+
+
+    /**
+     * OnCreateView that inflates and sets up the view
+     */
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+        final View DisplayFragmentView = inflater.inflate(R.layout.fragment_article,
```

Comment:
Please use the right cases for variable naming. This should read `displayFragmentView`